config/prod/htan2-crc-syn63298073.yaml

```diff
@@ -0,0 +1,19 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.8.3/templates/S3/synapse-external-bucket.j2"
+stack_name: "htan2-crc-syn63298073"
+stack_tags:
+  OwnerEmail: 'aditi.gopalan@sagebase.org'
+  CostCenter: "HTAN-DFCI / 120100"
```

Comment:
should be `DFCI HTAN / 120100` to match code at https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2553544733/Finance+Program+Codes+for+AWS+tagging

Reply:
changed, thank you!